src/applications/yellow-ribbon/sass/yellow-ribbon.scss

```diff
@@ -2,8 +2,20 @@
   .school-program {
     margin-top: 36px !important;
   }
+
+  .form-checkbox [type="checkbox"] {
+    top: 3rem;
+  }
 }
 
 .va-u-outline--none:focus {
   outline: none;
 }
+
+.form-checkbox {
+  position: relative;
+
+  & [type="checkbox"] {
+    cursor: pointer;
+  }
+}
```

Comment:
I'm still confused why we need any custom CSS. This structure doesn't work for YR? https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/user/profile/vet360/containers/ReceiveTextMessages.jsx#L96

Reply:
Oh nice! Let me see if adding that wrapping div works like they did!

Reply:
Hey! That worked! 🎉

Reply:
All we needed was just a wrapping `div` actually